test/assets/p-selectors.html

```diff
@@ -1,13 +1,14 @@
 <div id="a">hello <button id="b">world</button>
     <span id="f"></span>
-    <div id="c">
-        <template shadowrootmode="open">
-            shadow dom
-            <div id="d">
-                <template shadowrootmode="open">
-                    <a id="e">deep text</a>
-                </template>
-            </div>
-        </template>
-    </div>
-</div>
\ No newline at end of file
+    <div id="c"></div>
+</div>
```

Comment:
Firefox does not support the `<template shadowrootmode="open">` which is not a standart yet. 
Using the old way to insure we test it for it that makes their CDP implementation pass as well,